spec.html

```diff
@@ -27076,8 +27097,8 @@ <h1>Evaluate ( ): a Promise</h1>
             1. Else,
               1. Assert: _module_.[[Status]] is either ~evaluating-async~ or ~evaluated~.
               1. Assert: _module_.[[EvaluationError]] is ~empty~.
-              1. If _module_.[[AsyncEvaluation]] is *false*, then
-                1. Assert: _module_.[[Status]] is ~evaluated~.
+              1. If _module_.[[Status]] is ~evaluated~, then
+                1. Assert: _module_.[[AsyncEvaluationOrder]] is ~unset~.
```

Comment:
Am I correct in understanding that it's `~unset~` because being in this branch implies that evaluation completed synchronously, and so this module didn't participate in async evaluation at all? If so, possibly a NOTE to that effect would be helpful, as in

```suggestion
                1. NOTE: This implies that evaluation of _module_ completed synchronously.
                1. Assert: _module_.[[AsyncEvaluationOrder]] is ~unset~.
```

Reply:
Yes that's correct.